test_stdout_add__contains_no_garbage failure

On Python 3.7 with click 7.x, I encounter
```
[   31s] ______________ TemplateTests.test_stdout_add__contains_no_garbage ______________
[   31s] 
[   31s] self = <tests.test_main.TemplateTests testMethod=test_stdout_add__contains_no_garbage>
[   31s] 
[   31s]     def test_stdout_add__contains_no_garbage(self):
[   31s]         some_name = 'some_name_that_dont_exist'
[   31s]         with EvacuateCustomTemplatesDir(), \
[   31s]                 self.runner.isolated_filesystem():
[   31s]             # Define custom template
[   31s]             CUSTOM_DIR = os.path.expanduser('~/.gitignore_templates/')
[   31s]             os.makedirs(CUSTOM_DIR)
[   31s]             with open(CUSTOM_DIR + some_name + '.gitignore', 'w') as f:
[   31s]                 f.write('foobar')
[   31s]     
[   31s]             result = self.runner.invoke(cli, ['--stdout', '-a', 'foobar'])
[   31s]             stdout_str = result.output
[   31s]     
[   31s] >           self.assertEqual(stdout_str, 'foobar\n')
[   31s] E           AssertionError: '\nfoobar\n\n' != 'foobar\n'
[   31s] E           - 
[   31s] E             foobar
[   31s] E           -
```

--- a/git_ignore/main.py
+++ b/git_ignore/main.py
@@ -18,7 +18,7 @@ def echo_status(status, title, text, use_default_color=False):
     color = status_color[status]
     text_color = None if use_default_color else color
 
-    click.echo(click.style(f'[{status}]', fg=color) + f' {title}\n')
+    click.echo(click.style('[{}]'.format(status), fg=color) + ' {}\n'.format(title))
     if text:
         click.echo(
             click.style(tw.indent(tw.fill(text), '\t'), fg=text_color) + '\n'

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -79,7 +79,7 @@ class TemplateTests(unittest.TestCase):
             result = self.runner.invoke(cli, ['--stdout', '-a', 'foobar'])
             stdout_str = result.output
 
-            self.assertEqual(stdout_str, 'foobar\n')
+            self.assertEqual(stdout_str.strip(), 'foobar')
 
     def test_stdout__contains_no_garbage(self):
         some_name = 'some_name_that_dont_exist'

